kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/resolver/ResourceResolver.java

```diff
@@ -9,22 +11,22 @@
     static final String FILE_PREFIX = "file";
     static final String CLASSPATH_PREFIX = "classpath:";
 
-    public static URL resolve(String location) {
+    public static InputStream resolve(String content) {
```

Comment:
This method is used in https://github.com/arquillian/arquillian-cube/blob/master/kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/resources/KubernetesResourcesApplier.java#L102 which is expecting URL. You have to modify it to make it working as per modification